pyqrack/qrack_simulator.py

```diff
@@ -1026,6 +2344,16 @@ def _apply_pyzx_op(self, gate):
             raise RuntimeError("QrackSimulator C++ library raised exception.")
 
     def run_pyzx_gates(self, gates):
+        """PYZX Gates
+
+        Converts PYZX gates to circuit gates and appends them to the circuit.
```

Comment:
I understand the concept and terminology coming from an API like Qiskit, for this description, but it might be more accurate to say that Qrack starts immediately running the gate list in place upon the `QrackSimulator` state, rather than "appending them to the circuit."

For example, but feel free to tweak this: "Converts PYZX gates to `QrackSimulator` gate calls and immediately executes them."